package.json

```diff
@@ -16,20 +16,29 @@
   "devDependencies": {
     "babel-plugin-transform-remove-strict-mode": "^0.0.2",
     "babel-preset-latest": "^6.24.1",
-    "eslint-plugin-html": "^2.0.1",
-    "grunt": "^1.0.1",
+    "dojo": "1.13.0",
+    "dojox": "1.13.0",
+    "eslint-plugin-html": "4.0.3",
+    "grunt": "1.0.3",
     "grunt-babel": "^6.0.0",
     "grunt-bump": "^0.8.0",
     "grunt-contrib-connect": "^1.0.2",
-    "grunt-contrib-jasmine": "^1.0.0",
+    "grunt-contrib-jasmine": "2.0.1",
     "grunt-contrib-watch": "^1.0.0",
-    "grunt-eslint": "^19.0.0",
+    "grunt-eslint": "20.1.0",
     "grunt-if-missing": "^1",
-    "load-grunt-tasks": "^3.5.2",
+    "load-grunt-tasks": "4.0.0",
+    "napa": "3.0.0",
     "phantomjs-prebuilt": "^2.1.14"
   },
   "dojoBuild": "map-tools.profile.js",
   "scripts": {
+    "prepare": "napa simsalabim/jasmine-favicon-reporter",
     "test": "grunt travis --verbose"
+  },
+  "dependencies": {
+    "arcgis-js-api": "^4.7.x",
+    "grunt-sass": "2.1.0",
```

Comment:
is this a dependency or a dev dependency?

Reply:
Fixed in: 0428ad0